ocs_ci/utility/nfs_utils.py

```
@@ -850,3 +864,115 @@ def _get_nfs_server():
+    Args:
+        pod_obj: Pod object
+        file_path_in_pod (str): Path to file inside pod (e.g., /mnt/filename)
+
+    Returns:
+        int: Line count or 0 if failed
+    """
+    line_cmd = f"wc -l {file_path_in_pod}"
+    try:
+        pod_output = pod_obj.exec_cmd_on_pod(command=line_cmd, out_yaml_format=False)
+        return int(pod_output.split()[0])
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)